api/dbadapters/bigquery.ts

```diff
@@ -232,9 +232,26 @@ export class BigQueryDbAdapter implements IDbAdapter {
     return rows;
   }
 
-  public async schemas(database: string): Promise<string[]> {
-    const data = await this.getClient(database).getDatasets();
-    return data[0].map(dataset => dataset.id);
+  public async databases(): Promise<string[]> {
+    throw new Error("BigQuery does not support multiple databases in the same connection");
```

Comment:
i'm not sure this error message makes sense in the context of BQ.

it's not that it's got anything to do with the 'connection' - any connection can theoretically 'connect' to any BQ project, given the right permissions.

presumably we should just make the right API call to fetch all projects that this auth'd account has access to? https://cloud.google.com/bigquery/docs/reference/rest/v2/projects/list

Reply:
Ah I see. This is pretty big to do I think, because credentials aren't user scoped, and we don't have a system for setting that up at the moment.

For now I'll just make the error message more clear.